backend/alembic/versions/b8ce06a37505_create_ridership_table.py

```diff
@@ -17,32 +18,25 @@
 
 
 def upgrade() -> None:
-    op.execute(
-        """
-        CREATE TABLE IF NOT EXISTS public.ridership (
-	        id serial PRIMARY KEY,
-	        van_id int NOT NULL,
-	        route_id int NOT NULL,
-	        entered int NOT NULL,
-	        exited int NOT NULL,
-	        lat float8 NOT NULL,
-	        lon float8 NOT NULL,
-	        datetime timestamptz NOT NULL,
-	        UNIQUE (van_id, datetime),
-	        FOREIGN KEY (van_id)
-		        REFERENCES vans (id)
-		        ON DELETE CASCADE,
-	        FOREIGN KEY (route_id)
-		        REFERENCES routes (id)
-		        ON DELETE CASCADE
-        );
-    """
+    op.create_table(
+        "ridership",
+        sa.Column("id", sa.Integer, primary_key=True),
+        sa.Column("van_id", sa.Integer, nullable=False),
+        sa.Column("route_id", sa.Integer, nullable=False),
+        sa.Column("entered", sa.Integer, nullable=False),
+        sa.Column("exited", sa.Integer, nullable=False),
+        sa.Column("lat", sa.Float, nullable=False),
+        sa.Column("lon", sa.Float, nullable=False),
+        sa.Column("datetime", sa.DateTime(timezone=True), nullable=False),
+        sa.UniqueConstraint("van_id", "datetime"),
+        sa.ForeignKeyConstraint(["van_id"], ["vans.id"], ondelete="CASCADE"),
+        sa.ForeignKeyConstraint(["route_id"], ["routes.id"], ondelete="CASCADE"),
```

Comment:
Also, this is probably a longer discussion, but I doubt we ever want to delete analytics. @ezrichards thoughts?